src/api.ts

```diff
@@ -16,7 +15,6 @@ function get<T>(url: string): Promise<T> {
   // eslint-disable-next-line prefer-template
   const fullURL = BASE_URL + url + '.json';
```

Comment:
Consider using a template literal to construct the `fullURL` variable. This will allow you to remove the ESLint disable comment for the `prefer-template` rule.